tests/utils.py

```diff
@@ -75,3 +77,10 @@ def mock_reverse(view_name, args=None, kwargs=None, request=None, format=None):
 
 def fail_reverse(view_name, args=None, kwargs=None, request=None, format=None):
     raise NoReverseMatch()
+
+
+def remove_object_addresses(repr_str):
+    """
+    Remove object address hex strings from reprs, for stable comparison.
+    """
+    return re.sub(" at 0x[0-9A-F]+>", ">", repr_str)
```

Comment:
Perhaps bundle this up into something like `compat_repr` instead that does both the `remove_object_addresses` and `unicode_repr` aspects.